src/model/bpmn/shape/ShapeBpmnElementKind.ts

```diff
@@ -24,8 +24,21 @@ export enum ShapeBpmnElementKind {
   TASK = 'task',
   GATEWAY_PARALLEL = 'parallelGateway',
   GATEWAY_EXCLUSIVE = 'exclusiveGateway',
+  GATEWAY_INCLUSIVE = 'inclusiveGateway',
+  GATEWAY_COMPLEX = 'complexGateway',
   EVENT_START = 'startEvent',
   EVENT_END = 'endEvent',
   EVENT_INTERMEDIATE_CATCH = 'intermediateCatchEvent',
   EVENT_INTERMEDIATE_THROW = 'intermediateThrowEvent',
+  CALL_ACTIVITY = 'callActivity',
+  SUB_PROCESS = 'subProcess',
 }
+
+/**
+ * List supported ShapeBpmnElement
+ *
+ * Temporarily used until we support all ShapeBpmnElements
+ */
+export const supportedBpmnElementsKinds = Object.values(ShapeBpmnElementKind).filter(kind => {
```

Comment:
:warning: only required because we added some probably not needed bpmn element kind

Reply:
As we discussed, I'll remove it